coverage/coverage.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+
+coverage erase
+
+find * \( -name '*.py' -o -name '*.pyi' \) -and -not -path './venv/*' \
+ -and -not -path './stubs/*' -exec python3 -m compileall -q -j 0 {} +
+
+
+run_coverage() {
+    python -m pytest -xvv $1 --cov --cov-append
+}
+export -f run_coverage
+
+for CUR in $(find 'tests' \( -name '*.py' -and -name 'test_*' \) \
+            -and -not -path 'tests/data/*' \
+            -and -not -path 'tests/__pycache__/*' |
+            sort -sf); do
+    run_coverage ${CUR}
+done
+
+coverage report
```

Comment:
are the coverages properly merging?

Reply:
check `Run Tests Coverage` section in https://github.com/Accern/accern-data-client/runs/6402612860?check_suite_focus=true

Reply:
what am I supposed to see there? I don't see a mention of the different reports being properly combined